Controller/MediaAdminController.php

```diff
@@ -109,6 +117,126 @@ public function listAction(Request $request = null)
     }
 
     /**
+     * @param Request $request
+     *
+     * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException|\LogicException
+     *
+     * @return \Symfony\Bundle\FrameworkBundle\Controller\Response|\Symfony\Component\HttpFoundation\Response
+     */
+    public function multiUploadAction(Request $request)
```

Comment:
IMO not, as the other actions like `list` are not final right now, or do we keep this because of BC?

@greg0ire WDYT? final or not?